app.js

```diff
@@ -0,0 +1,45 @@
+const createError = require('http-errors');
+const express = require('express');
+const app = express();
+const requestRobotsTxtFiles = require('./services/robots.service');
+const gzipArrs = require('./services/gzip.service');
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.get('/', (req, res) => {
+  res.json('Welcome to the express app')
+});
+
+app.get('/site-urls/:lookUpUrl', async (req, res, next) => {
+
+  const scrapeRobotsTxtFiles = async (lookUpUrl) => {
+    robotTxtFileUrl = `https://${lookUpUrl}/robots.txt`;
+    const urlArr = await requestRobotsTxtFiles(robotTxtFileUrl);
+    return urlArr;
+  };
+
+
+  const lookUpUrl = req.params.lookUpUrl;
+  const siteMapArr = await scrapeRobotsTxtFiles(lookUpUrl);
+  const nestedSitemapUrlArr = await gzipArrs(siteMapArr); 
```

Comment:
what do we do with the result?

Reply:
I am going to scrape urls from the sitemaps returned as the result

Reply:
but in this PR this result is not used... 
I don't think it's a good practice to leave unused code cause eventually it will be hard to maintain, but if you really want to then add at least a TODO comment describing why the unused code is there